packages/chakra-ui-docs/docs/tooltip.mdx

```diff
@@ -1 +1,118 @@
+
+<LiveEditor>
+  {`
+<c-tooltip has-arrow label="Sends your order to Delivery hero" placement="top" bg="blue.600">
+   <c-button>Place Order</c-button>
+</c-tooltip>
+`}
+</LiveEditor>
+
+### Placement
+
+Using the `placement` prop you can adjust where your tooltip will be displayed.
+
+<LiveEditor>
+  {`
+<c-stack should-wrap-children>
+    <c-tooltip has-arrow label="left" placement="left">
+        <c-button variant="outline">Left</c-button>
+    </c-tooltip>
+     <c-tooltip has-arrow label="top" placement="top">
+        <c-button variant="outline">Top</c-button>
+    </c-tooltip>
+     <c-tooltip has-arrow label="right" placement="right">
+        <c-button variant="outline">Right</c-button>
+    </c-tooltip>
+     <c-tooltip has-arrow label="bottom" placement="bottom">
+        <c-button variant="outline">Bottom</c-button>
+    </c-tooltip>
+</c-stack>
+`}
+</LiveEditor>
+
+### Props
+
+| Name                 | Type                 | Default  | Description                                                                           |
+| -------------------- | -------------------- | -------- | ------------------------------------------------------------------------------------- |
+| `isOpen`             | `boolean`            |          | If `true`, the tooltip is shown.                                                      |
+| `defaultIsOpen`      | `boolean`            |          | If `true`, the tooltip is initially shown.                                            |
+| `label`              | `string`             |          | The label of the tooltip.                                                             |
+| `aria-label`         | `string`             |          | An alternate label for screen readers.                                                |
+| `placement`          | `PopperJS.Placement` | `bottom` | Position the tooltip relative to the trigger element as well as surrounding elements. |
+| `children`           | `Vue.VNode`          |          | The `VNode` to be used as the trigger of the tooltip.                                 |
+| `hasArrow`           | `boolean`            |          | If `true` display an arrow tip on the tooltip.                                        |
+| `showDelay`          | `number`             |          | The delay in `ms` for the tooltip to show                                             |
+| `hideDelay`          | `number`             |          | The delay in `ms` for the tooltip to hide                                             |
+| `closeOnClick`       | `boolean`            |          | If `true` hide the tooltip, when the trigger is clicked.                              |
+| `shouldWrapChildren` | `boolean`            |          | If `true`, the tooltip will wrap the children in a `span` with `tabIndex=0`           |
+
+### Events
+
+| Name     | Type       | Default | Description                             |
```

Comment:
Nice work with this Kelvin.

For the Events table instead of type, let's use `Payload` type which is the type of the data that the event emits. In this case, we don't emit anything for the `CTooltip` so we can just put a dash here indicating nothing. (`-`)/

We can also drop the **Default** column from the events table.
Otherwise nice work 👍

Reply:
Okay